.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,90 @@
+name: Benchmark GTSFM on select datasets
+on: [pull_request]
+jobs:
+  door-dataset-sift:
```

Comment:
needs new name since not only door dataset anymore?

Reply:
changed